packages/tasks/src/snippets/inputs.ts

```diff
@@ -79,6 +79,9 @@ const inputsTextToAudio = () => `"liquid drum and bass, atmospheric synths, airy
 
 const inputsAutomaticSpeechRecognition = () => `"sample1.flac"`;
 
+const inputsTabularPrediction = () =>
+	`'{"Height":[11.52,12.48],"Length1":[23.2,24.0],"Length2":[25.4,26.3],"Species": ["Bream","Bream"]}'`;
```

Comment:
I think this would be redundant: the user cannot to inference with a random model on Hugging Face Hub because the features are specific for that model, so they can technically only infer with the model they've trained (or at least they have the dataset on hand already, they know the columns). The dataset on model card is generated by sampling the training data anyway, one doesn't need to infer with it.

Reply:
So in a way this sample only shows how they should structure their dataframes to put to payload

Reply:
I'm not sure I understand your reply. 

My suggestion was that at one point, we should use the content from https://huggingface.co/scikit-learn/Fish-Weight/blob/main/README.md?code=true#L8-L33 . Right now we'll consistently show the same columns (height, length1, length2, ...) for all repos, meaning we show widgets that will not work when they try them. 

> the dataset on model card is generated by sampling the training data anyway, one doesn't need to infer with

Do you mean the model card metadata widget example or a dataset?


(in any case, IMO this PR is good to be merged as is - we can revisit this if/once we have thousands of sklearn repos)